external_search.py

```diff
@@ -32,6 +32,8 @@ def reset(cls):
     def __init__(self, url=None, works_index=None):
     
         self.log = logging.getLogger("External search index")
+        self.works_index = None
+        self.works_alias = None
```

Comment:
This stops a crash in a test scenario where the app server is started up without any search configuration.